README.md

```diff
@@ -1 +1,966 @@
+port:9090
+};
+
+@http:ServiceConfig {basePath:"/phonestore"}
+service<http:Service> phone_store_service bind listener {
+``` 
+
+- Here we have used ``  @kubernetes:Deployment `` to specify the Docker image name which will be created as part of building this service. 
+- We have also specified `` @kubernetes:Service `` so that it will create a Kubernetes service, which will expose the Ballerina service that is running on a Pod.  
+- In addition we have used `` @kubernetes:Ingress ``, which is the external interface to access your service (with path `` /`` and host name ``ballerina.guides.io``)
+
+- Now you can build a Ballerina executable archive (.balx) of the service that we developed above, using the following command. This will also create the corresponding Docker image and the Kubernetes artifacts using the Kubernetes annotations that you have configured above.
+  
+```
+   $ ballerina build 
+   
+   @kubernetes:Service                      - complete 1/1
+   @kubernetes:Ingress                      - complete 1/1
+   @kubernetes:Docker                       - complete 3/3 
+   @kubernetes:Deployment                   - complete 1/1
+  
+   Run following command to deploy kubernetes artifacts:  
+   kubectl apply -f ./target/phone_store_service/kubernetes
+   
+   @kubernetes:Service                      - complete 1/1
+   @kubernetes:Ingress                      - complete 1/1
+   @kubernetes:Docker                       - complete 3/3 
+   @kubernetes:Deployment                   - complete 1/1
+  
+   Run following command to deploy kubernetes artifacts:  
+   kubectl apply -f ./target/phone_order_delivery_service/kubernetes
+  
+   
+```
+
+- You can verify that the Docker image that we specified in `` @kubernetes:Deployment `` is created, by using `` docker images ``. 
+- Also the Kubernetes artifacts related our service, will be generated under `` ./target/phone_store_service/kubernetes`` and ``/target/phone_order_delivery_service/kubernetes``. 
+- Now you can create the Kubernetes deployment using:
+
+```bash
+   $ kubectl apply -f ./target/phone_store_service/kubernetes
+   
+   deployment.extensions "ballerina-guides-phone_store_service" created
+   ingress.extensions "ballerina-guides-phone_store_service" created
+   service "ballerina-guides-phone_store_service" created
+   
+   kubectl apply -f ./target/phone_order_delivery_service/kubernetes
+   
+   deployment.extensions "ballerina-guides-phone_order_delivery_service" created
+   ingress.extensions "ballerina-guides-phone_order_delivery_service" created
+   service "ballerina-guides-phone_order_delivery_service" created
+```
+
+- You can verify Kubernetes deployment, service and ingress are running properly, by using following Kubernetes commands. 
+
+```bash
+   $ kubectl get service
+   $ kubectl get deploy
+   $ kubectl get pods
+   $ kubectl get ingress
+```
+
+- If everything is successfully deployed, you can invoke the service either via Node port or ingress. 
```

Comment:
Shall we rephrase this to "If all artifacts are successfully deployed, you can invoke the service either via Node Port or Ingress."? Refer to https://github.com/ballerina-guides/restful-service/blob/master/README.md#deploying-on-kubernetes as this was reviewed and refined by the doc team.